### ISSUE TITLE
Are these libraries compatable with KiCAD 5.x?

### ISSUE THREAD

**Issue by Sonophoto** [digikey/digikey-kicad-library#73]
MEA CULPA: Not sure if this is the right way/place to ask but I'm on github and this question seemed likely to come up anyway.

Perhaps add some info on versioning to the [README.md](https://github.com/digikey/digikey-kicad-library/blob/master/README.md)?

**Comment by Reid-L** [comment]
@Sonophoto 

Noli sollicitari: Thanks for the inquiry and the feedback.

As far as we are aware, through our testing, the libraries are compatible with KiCad 5.x. Most libraries created in KiCad 4.0.7 should be compatible anyway.

We will do what we can to get some versioning and compatibility information in the README. Thanks for the feedback.

Thanks,
Reid L.
Digi-Key Electronics

**Comment by Sonophoto** [comment]
update: if you do add something to the readme.md this might be a good link to add to it:

https://forum.digikey.com/c/kicad

**Comment by mariela1522** [comment]
Hi, how can this library to kicad? im new sorry

**Comment by jacky4566** [comment]
Yes I have been using this library in 5.0 no problem. To import:

For the Symbols:
Open the Symbol Library editor>Preferences>Manage Symbol Libraries>Select "Browse Libraries"
Then select all the dk_xxx.lib files.

For the Footprints:
Follow the same as above and select the .pretty folder.

**Comment by poeschlr** [comment]
I can assure anybody wondering about this that any library created in KiCad version 4 is fully compatible with KiCad version 5. (This holds true only for libraries! Schematics need to be ported from version 4 to version 5.)

----

The other way round is not guaranteed. KiCad 5 supports more features for both symbols and footprints. If any symbol or footprint within a given library uses any of the new features then the whole library can not be used in version 4.
